pkg/apis/crds/karpenter.k8s.aws_ec2nodeclasses.yaml

```diff
@@ -221,7 +221,8 @@ spec:
                     type: string
                 type: object
               role:
-                description: Role is the AWS identity that nodes use.
+                description: Role is the AWS identity that nodes use. This field is
```

Comment:
Can we add something in the docs/definition here that explains why we chose to make this immutable? I want to reduce the questions we get about this.